attack/scripts/bashrc

```diff
@@ -86,6 +86,21 @@ find_current_task() {
 }
 readonly -f find_current_task
 
+check_if_task_set() {
+  local task
+  task=$(jq -r '.current_task' /progress.json)
+  if [[ ${task} = "null" ]]; then
+    echo "You have not started a task yet. One can be set with 'start_task ...'. Would you like to continue anyway without starting a task? Enter '1' or '2'"
```

Comment:
I think you mean enter yes/no as that is what the following case statements looks for

Reply:
+1

Reply:
This is using the `select` command and it gives the user an option like this:

```
root@attack:/attack[0]$ starting_point 
You have not started a task yet. One can be set with 'start_task ...'. Would you like to continue anyway without starting a task? Enter '1' or '2'
1) Yes
2) No
#? 
```
So you need to actually enter a number. If you don't it won't work:
```
root@attack:/attack[1]$ starting_point 
You have not started a task yet. One can be set with 'start_task ...'. Would you like to continue anyway without starting a task? Enter '1' or '2'
1) Yes
2) No
#? Yes
#? No
#? y
#? yes
#? no
#? 2
root@attack:/attack[1]$
```